source/geod24/RingBuffer.d

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************************
+
+    Contains a ring buffer implementation with fixed size
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module geod24.RingBuffer;
+
+import std.range;
+
+/*******************************************************************************
+
+    A ring buffer with fixed size
+
+    Params:
+        T = Type of elements
+
+*******************************************************************************/
+
+class RingBuffer (T)
```

Comment:
public please. :) And then feel free to merge!

Reply:
Done. But, I dont have write access here.